app/views/catalog/_show_sidebar.html.erb

```diff
@@ -3,7 +3,8 @@
 
 <div class="sidebar-buttons">
 <%= render :partial => "show_web_services" %>
-<%= render :partial => "show_downloads" %>
+<%= render(Geoblacklight::DownloadLinksComponent.new(document:)) if document_downloadable? %>
```

Comment:
why is `document_downloadable?` here instead of in the `render?` method in the component?  because it is defined in `GeoblacklightHelper` helper and includes a call to `user_signed_in?`, a helper method likely in Devise or elsewhere?  

which was causing me grief in testing -- i will look some more